File: C/100 C codes/gcd.c
#include<stdio.h>

int main() {
    int num1, num2, gcd;
    
    
    printf("Enter the first number\n");
    scanf("%d", &num1);
    printf("Enter the second number\n");
    scanf("%d", &num2);
    
    
    while (num2 != 0) {
        int temp = num2;
        num2 = num1 % num2;
        num1 = temp;
    }
    
    gcd = num1;
    printf("GCD is: %d\n", gcd);
    
    return 0;
}

File: Python/Python/C codes in py/gcd.py
num1=int(input("Enter number1:"))
num2=int(input("Enter number2:"))
min= num1 if num1<num2 else num2
gcd=min
while(num1%min!=0 or num2%min!=0):
    min-=1
    gcd=min
print(f"GCD is {gcd}")  